src/main/java/net/pms/database/MediaTableTVSeries.java

```diff
@@ -325,7 +325,7 @@ public static Long set(final Connection connection, final JsonObject tvSeries, f
 						} else {
 							insertQuery = SQL_INSERT_ALL;
 						}
-						try (PreparedStatement insertStatement = connection.prepareStatement(insertQuery, PreparedStatement.RETURN_GENERATED_KEYS)) {
+						try (PreparedStatement insertStatement = connection.prepareStatement(insertQuery, Statement.RETURN_GENERATED_KEYS)) {
```

Comment:
Why?

Reply:
In the interest of code clarity, static members of a base class should never be accessed using a derived type's name.

Doing so is confusing and could create the illusion that two different static members exist.